Add faucet guide - Closes #1153 (#1226)
LiskHQ/lisk-docs: modules/ROOT/pages/guides/app-development/faucet.adoc

= Enabling a faucet for a devnet or testnet
Mona Bärenfänger <mona@lightcurve.io>
// Settings
:toc: preamble
:imagesdir: ../../../assets/images
:idprefix:
:idseparator: -
:sectnums:
:experimental:
// URLs
:url_recaptcha_keys: https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do
:url_faucet: http://localhost:4004
:url_dashboard: http://localhost:4005
// Project URLS
:url_plugin_faucet: references/lisk-framework/faucet-plugin.adoc
:url_guide_dashboard: guides/app-development/dashboard.adoc
:url_guide_genesisblock: guides/app-development/genesis-block.adoc

The xref:{url_plugin_faucet}[] provides a user interface in the browser to receive a certain amount of tokens on request.

> The Faucet plugin is a useful tool during development and also in testing networks.

Author: ```suggestion
The Faucet plugin is a useful tool to use in devnets or testnets.
```
